platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/gossip/shadowgraph/Shadowgraph.java

```diff
@@ -571,7 +571,7 @@ public synchronized boolean addEvent(@NonNull final PlatformEvent event) throws
                 return true;
             } else {
                 if (status == InsertableStatus.EXPIRED_EVENT) {
-                    logger.warn(
+                    logger.error(
                             SYNC_INFO.getMarker(),
```

Comment:
I think you need to make this the `EXCEPTION` marker, otherwise we'll never see this (`SYNC_INFO` logging isn't turned on anywhere AFAIK)

Reply:
done